internal/sdkv2provider/resource_cloudflare_access_group_test.go

```diff
@@ -21,53 +19,6 @@ func init() {
 	})
 }
 
-func testSweepCloudflareAccessGroups(r string) error {
```

Comment:
did you intend to remove this?

Reply:
Yes - I saw it was unused and was just doing some cleanup, can revert it if we want

Reply:
ah, this should be used by the sweeper above. let me fix that.